app/controllers/scholarships_controller.rb

```diff
@@ -29,54 +29,55 @@ def update_imported_user(file)
     (2..spreadsheet.last_row).each do |i|
       row = Hash[[header, spreadsheet.row(i)].transpose]
       puts(header)
-      @scholarship = Scholarship.new()
-      @user = User.new(
+      user = User.new(
         email: row['CORREO'],
         :password => '111111',
```

Comment:
Should we make it more secure by setting the password as the CVU?

Reply:
I think that we could discuss this with the client and do what it is best from their perspective. For now I think that we can leave as it is.